src/helpers/generate-matrix.ts

```diff
@@ -17,11 +19,42 @@ import { chunk } from 'lodash';
 export class GenerateMatrix extends HelperInputs {
   paths = '';
   batches?: string;
+  load_balancing_sizes?: string;
 }
 
-export const generateMatrix = ({ paths, batches = '1' }: GenerateMatrix) => {
+export const generateMatrix = ({ paths, batches: _batches = '1', load_balancing_sizes }: GenerateMatrix) => {
   const matrixValues = paths.split(/[\n,]/);
-  return {
-    include: chunk(matrixValues, Math.ceil(matrixValues.length / Number(batches))).map(chunk => ({ path: chunk.join(',') }))
-  };
+  const batches = Number(_batches);
+  if (!load_balancing_sizes) {
+    return {
+      include: chunk(matrixValues, Math.ceil(matrixValues.length / batches)).map(chunk => ({ path: chunk.join(',') }))
+    };
+  } else {
```

Comment:
Unnecessary else block here, can just use the [guard clause pattern](https://deviq.com/design-patterns/guard-clause)